### PR TITLE
Fix mpa seg export issue

### PR DESCRIPTION
**Pull request by chuneuny-emily**
This PR fixes export issue with mpa model templates.

### CODE REVIEW

File: mpa/seg/stage.py [hunk 1]
```diff
@@ -197,9 +197,10 @@ def configure_task_classes(self, cfg, task_adapt_op):
 
         # Model classes
         if task_adapt_op == 'REPLACE':
-            if len(data_classes) == 0:
-                raise ValueError('Data classes should contain at least one class!')
-            model_classes = data_classes.copy()
+            if len(data_classes) == 1:
+                model_classes = org_model_classes.copy()
+            else:
+                model_classes = data_classes.copy()
```
Comment (goodsong81) [inline mpa/seg/stage.py]: What do you mean by "== 1"?

Comment (chuneuny-emily) [reply mpa/seg/stage.py]: always adding bg. 1 is bg size.

Comment (goodsong81) [reply mpa/seg/stage.py]: Thanks. Please add some comments for understanding.

**Review by JihwanEom — APPROVED** [review]
LGTM, please check flake8 convention for your comment

**Pull request merged**